media-fonts/joypixels/joypixels-6.5.0.ebuild

```
@@ -20,7 +20,16 @@ RESTRICT="bindist mirror"
 
 FONT_CONF=( "${FILESDIR}"/99-joypixels.conf )
 
+DOC_CONTENTS="Free for personal/education use only,
+premium/enterprise license required for any other use:
+    https://www.joypixels.com/licenses"
+
 src_prepare() {
 	default
 	cp "${DISTDIR}"/${P}.ttf "${S}"/${P}.ttf || die
 }
+
+src_install() {
+	font_src_install
+	readme.gentoo_create_doc
+}
```

```suggestion
}

pkg_postinst() {
	readme.gentoo_print_elog
}
```
(I thought it'd be on by default, but it's not)
I'll add it, thanks!

---

thanks!